index.html

```diff
@@ -0,0 +1,291 @@
+                />
+              </div>
+              <span>Lecture</span>
+              <p>
+                Listen to the speakers from various countries about the messages
+                of sharing and opening
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/forum.svg"
+                  alt="forum"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Exhibition</span>
+              <p>
+                Appreciate various creations applying CC license of participants
+                from this community
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/ccIgnite.svg"
+                  alt="ccIgnite"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Forum</span>
+              <p>
+                Have the time to share your thoughts and opinionswith experts
+                for each topic
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>Workshop</span>
+              <p>
+                Denotaing and small community using open sources and license
+                than sitting
+              </p>
+            </div>
+            <div class="sectiontwo__container__items__item">
+              <div class="sectiontwo__container__items__item__responsive">
+                <img
+                  src="static/imgs/workshop.svg"
+                  alt="workshop"
+                  class="responsive sectiontwo__container__items__item__responsive__respo"
+                />
+              </div>
+              <span>CC Ignite </span>
+              <p>
+                Get opportunity to network the CC affiliates around the world,
+                also after conference
+              </p>
+            </div>
+          </div>
+          <a class="sectiontwo__container__title display" href="#" id="first"
+            >SEE THE WHOLE PROGRAM</a
+          >
+          <a class="sectiontwo__container__title" href="#" id="second">
+            Join unicef's annual human support</a
+          >
+        </div>
+      </section>
+      <!-- end sectiontwo -->
+      <!-- start sectionthree -->
```

Comment:
- [ Optional] to make your code clean, please make sure that you remove unnecessary comments. You should add comments in codes only if you have a specific reason for you to put them in the codebase.